Fix compilation with gcc 4.6 by correcting the way that linker options
are passed to gcc.

Signed-off-by: Sven Neumann <s.neumann@raumfeld.com>
--- kexec-2.0.2/purgatory/Makefile.orig	2011-09-26 14:16:44.256595912 +0200
+++ kexec-2.0.2/purgatory/Makefile	2011-09-26 14:17:01.606621357 +0200
@@ -56,7 +56,7 @@
 			-I$(srcdir)/include \
 			-I$(shell $(CC) -print-file-name=include)
 $(PURGATORY): LDFLAGS=$($(ARCH)_PURGATORY_EXTRA_CFLAGS)\
-			--no-undefined -nostartfiles -nostdlib -nodefaultlibs \
+			-Wl,-no-undefined -nostartfiles -nostdlib -nodefaultlibs \
 			-e purgatory_start -r
 
 $(PURGATORY): $(PURGATORY_OBJS)
